tests/test_cli.py

```
@@ -13,82 +13,103 @@ def cli_runner():
 
 
 @pytest.fixture(autouse=True)
+def log_level(caplog):
+    caplog.set_level(logging.DEBUG)
+
+
+@pytest.fixture
 def default_loader(mocker):
     return mocker.patch("little_cheesemonger._cli.default_loader")
 
 
+@pytest.fixture
+def import_loader_function(mocker):
+    return mocker.patch("little_cheesemonger._cli.import_loader_function")
+
+
 @pytest.fixture(autouse=True)
 def run(mocker):
     return mocker.patch("little_cheesemonger._cli.run")
 
 
 def test_process_kwargs__return_dict():
-    assert _process_kwargs(("foo=bar", "baz=qux")) == {"foo": "bar", "baz": "qux"}
+    assert process_kwargs(("foo=bar", "baz=qux")) == {"foo": "bar", "baz": "qux"}
 
 
 def test_process_kwargs__malformed_mapping_string__raise_LittleCheesemongerError():
     with pytest.raises(LittleCheesemongerError):
-        _process_kwargs(("invalid",))
+        process_kwargs(("invalid",))
 
 
-def test_entrypoint__no_args(cli_runner):
-    result = cli_runner.invoke(entrypoint)
+def test_entrypoint__no_args(cli_runner, default_loader):
+    result = cli_runner.invoke(entrypoint, catch_exceptions=False)
```

`catch_exceptions` is required here to make the test not swallow unhandled exceptions